resources/js/Pages/Admin/Rooms/UpdateRoomForm.vue

```diff
@@ -159,6 +186,8 @@ export default {
             this.form.floor = room?.floor;
             this.form.building = room?.building;
             this.form.status = room?.status;
+            this.form.status = room?.min_days_advance;
+            this.form.status = room?.max_days_advance;
```

Comment:
Should match the min and max variables in the form

Reply:
Addressed, thanks for catching that.